mvn-build-all.sh

```diff
@@ -10,17 +10,31 @@ fi
 
 echo "[INFO] Start a build."
```

Comment:
- [ ] Are `mvn-build-all.sh` and `.bat` really necessary? I think that they will be unnecessary if include the `terasoluna-gfw-dependencies` to modules of root pom.